.gitignore

```diff
@@ -23,3 +23,5 @@ hs_err_pid*
 *.iml
 **/.gradle
 build
+
+src/gen
```

Comment:
note: This directory is no-longer generated, which is why I removed it from the `.gitignore`. However, it probably makes sense to keep the ignore-rule for some more time, as it might be confusing when you temporarily checkout an older commit and then have all these untracked files afterward.